gensim/matutils.py

```diff
@@ -667,45 +667,54 @@ def ret_log_normalize_vec(vec, axis=1):
 
 def unitvec(vec, norm='l2'):
     """Scale a vector to unit length.
-
+    
```

Comment:
please don't make unrelated changes, this empty line(s) is correct by docstring convention.